app/vmselect/netstorage/netstorage.go

```diff
@@ -2946,6 +2950,31 @@ func getStorageNodes() []*storageNode {
 	return snb.sns
 }
 
+var (
+	nodeID     uint64
+	nodeIDOnce sync.Once
+)
+
+// GetNodeID returns unique identifier for underlying storage nodes.
+func GetNodeID() uint64 {
```

Comment:
it's better to return always 0 value for consistency. Multi level cluster setup is not intended to use with persistent storage node ids.